.github/workflows/release-published.yml

```diff
@@ -0,0 +1,20 @@
+name: Checks after release is published
+on:
+  release:
+    types: ['published']
+
+jobs:
+  validate-github-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: k14s/setup-k14s-action@v1
```

Comment:
Might be nice to configure this action to only install `ytt`?  I think by default you get everything.

```
- uses: k14s/setup-k14s-action@v1
  with:
    only: ytt
```

Docs: https://github.com/k14s/setup-k14s-action#usage

Reply:
This file is not supposed to be part of this PR :( going to remove it, but add the suggestion to the PR #274